### DISCUSSION TITLE
切换路由,改变变量导致播放停止重置,找到解决办法了

### DISCUSSION THREAD

**Discussion by lxhlxh520** [SevenOutman/vue-aplayer#248] [General]
`abc.vue`
```javascript
<aplayer
        v-if="musictoplay.src"
        ref="aplayer"
        float
        :listFolded="true"
        :volume="0.3"
        repeat="list"
        :music="musictoplay"
        :list="musiclist"
      />
```
用v-if是为了手动控制aplayer渲染时机, 防止初始化的时候发送垃圾请求导致报错,我自己是在mounted的时候赋值music对象,实测效果不错.
最重要一点:   服务器响应的header中必须要包括'Accept-Ranges':'bytes',  chrome浏览器看不到这个头就会在导致拖动播放进度的时候强制重新播放

**Comment by SevenOutman** [comment]
Hi @lxhlxh520 ，感谢反馈。看上去这不是一个问题，我将会将此 Issue 转换为 Discussion

**Comment by lxhlxh520** [comment]
调试了几个小时,最终方案确定
不要使用v-once指令,会使组件丢失reactivity,   罪魁祸首是chrome浏览器, 服务器响应的header中必须要包括'Accept-Ranges':'bytes',加了这玩意后一切都正常了
